src/components/atoms/card/Card.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../styles/2-variables/_variables-index';
+
+.card-container {
+	position: relative;
```

Comment:
Heeft deze een functie? Wellicht zie ik iets over het hoofd, komt er straks iets in te zitten dat `absolute` gepositioneerd moet worden?